### ISSUE TITLE
Sharing password input for visitors is wrongly styled

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/server#5560]
### 2FA style
> ![nextcloud_-_2017-06-29_13 07 13](https://user-images.githubusercontent.com/213943/27684860-09a95874-5ccc-11e7-9e6a-bcfbe229c16c.png)

1. Has background box ✅ 
2. Has white font ✅ 

### Sharing password
> ![nextcloud_-_2017-06-29_13 07 27](https://user-images.githubusercontent.com/213943/27684887-2a34a92c-5ccc-11e7-808e-522c5e2bc9e2.png)

1. Missing the box ❌ 
2. Black font not readable on the background ❌ 

@jancborchardt can you have a look? The template is at https://github.com/nextcloud/server/blob/740659a04c28f8b2bfd9265d459dd3bca202300f/apps/files_sharing/templates/authenticate.php#L7-L26

**Comment by jancborchardt** [comment]
Good catch, pull request at https://github.com/nextcloud/server/pull/5562 :)
